components/Shared/AccountCard/index.jsx

```diff
@@ -1,14 +1,17 @@
 import React, { forwardRef } from 'react'
+import styled from 'styled-components'
 import { string, func, bool } from 'prop-types'
 import { ADDRESS_PROPTYPE } from '../../../customPropTypes'
 import Box from '../Box'
 import Glyph from '../Glyph'
 import Button from '../Button'
 import { ButtonCopyAccountAddress } from '../IconButtons'
-import { BigTitle, Text } from '../Typography'
+import { BigTitle, Text, Title } from '../Typography'
 import truncate from '../../../utils/truncateAddress'
 import copyToClipboard from '../../../utils/copyToClipboard'
 
+const AccountCardAddress = styled(Title)``
```

Comment:
Let's save ourselves the variable here and just use `Title` inline, passing it the necessary props.